plopfile.js

```diff
@@ -50,17 +50,15 @@ module.exports = function (plop) {
         const indexFile = `${plop.getDestBasePath()}/packages/components/src/index.ts`;
 
         if (fs.existsSync(indexFile)) {
-          const nameRegex = /default as ([^\s]*)/;
-          const sorted = fs
-            .readFileSync(indexFile, "utf8")
-            .split(";\n")
-            .sort((a, b) => {
-              const aName = a.match(nameRegex)[1];
-              const bName = b.match(nameRegex)[1];
-              return aName.localeCompare(bName);
-            })
-            .join(";\n");
-          fs.writeFileSync(indexFile, sorted + ";\n");
+          // Split the index file into lines.
+          const lines = fs.readFileSync(indexFile, "utf8").split("\n");
+
+          // Sort each line.
+          const sorted = lines.sort((a, b) => a.localeCompare(b)).join("\n");
```

Comment:
Instead of extracting the exported name or path, I decided to sort by the entire line. End result is we get 2 groups: the components and the types

```ts
export a from 'a';
export b from 'b';
export c from 'c';
export type b from 'b';
export type c from 'c';
```

I think that's fine for our purposes.

Reply:
I'm good with this! is consistent & predictable 👍 thanks for noticing & fixing this issue